content/docs/user-guide/basic-concepts/stage.md

```diff
@@ -0,0 +1,8 @@
+---
+name: 'Stage'
+match: ['stage', 'stages']
+tooltip: >-
+  A single step in a pipeline which may import data, run experiments and produce
+  some (intermediate or final) output. Stages may depend other stages and they
+  can be automatically invalidated when their dependencies change.
```

Comment:
Good start. I'd look in https://dvc.org/doc/command-reference/run for the existing definition/explanation. Specific questions on this current text:
- Stages run experiments? I think not exactly
- Probably need to describe "invalidated"

Reply:
A user runs experiments _in stages_ and a stage may be completely about running an experiment, right? Am I missing something? Stages don't run experiment but a stage may only be about running an experiment. 

Will add a "user" there. 

Instead of invalidation, I can tell how `run` or `repro` skips some stages but succinctly using the "invalidation" seemed more appropriate.

Reply:
Experiments are broader than individual stages (see WIP https://dvc-org-guide-experimen-xigcev.herokuapp.com/doc/user-guide/experiment-management). For now I think there's no nee to complicate this tooltip with the concept of experiments.

> Instead of invalidation, I can tell how run or repro skips some stages

Check the parameter and pipelines file tooltips: they use `invalid (considered outdated)`